README.md

```diff
@@ -33,7 +33,7 @@ For installation of Jekyll please follow https://jekyllrb.com/docs/installation/
 
 ### How to Update gems
 
-bundle update github-pages
+Delete Gemfile.lock and run bundle install
```

Comment:
Isn't bundle install for fresh installations?  Does deleting the lock and running update work?

Reply:
'bundle update github-pages' doesn't work when gemfile.lock doesn't exist, says you need to do 'bundle install'

Reply:
In a normal Ruby project, we'd commit the Gemfile.lock into the repo so that everyone who contributes uses the same dependency versions. 

We're relying on the `github-pages` gem's version constraints instead for this repo, so the lock file isn't important. New contributors run `bundle install`. If Github updates their gem, people who already installed it can run `bundle update` or delete the lock file as written. It shouldn't be necessary very often.